parser/src/main/kotlin/com/linkedin/dex/parser/DecodedValue.kt

```diff
@@ -24,14 +24,21 @@ sealed class DecodedValue {
     data class DecodedType(val value: String) : DecodedValue()
     object DecodedNull : DecodedValue()
     data class DecodedBoolean(val value: Boolean) : DecodedValue()
+    data class DecodedEnum(val value: String) : DecodedValue()
     // TODO: DecodedType
     // TODO: DecodedField
     // TODO: DecodedMethod
-    // TODO: DecodedEnum
     // TODO: DecodedArrayValue
     // TODO: DecodedAnnotationValue
-    
+
     companion object {
+        fun readStringInPosition(dexFile: DexFile, position: Int): String {
```

Comment:
Thanks for extracting common code into this new function!

Reply:
NP: this should probably be a private function, since its only used internally for parsing

Reply:
Fixed